src/components/structures/RoomView.tsx

```diff
@@ -974,9 +974,8 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
 
         CHAT_EFFECTS.forEach(effect => {
             if (containsEmoji(ev.getContent(), effect.emojis) || ev.getContent().msgtype === effect.msgType) {
-                // For initial threads launch, chat effects are disabled
-                // see #19731
-                if (!SettingsStore.getValue("feature_thread") || !ev.isThreadRelation) {
+                // For initial threads launch, chat effects are disabled see #19731
+                if (!SettingsStore.getValue("feature_thread") || !ev.isRelation(THREAD_RELATION_TYPE.name)) {
```

Comment:
we should be checking both `name` and `altName`, just give up on the namespace entirely, or document why we don't care about the `altName` with code comments.

Reply:
`THREAD_RELATION_TYPE` is a `ServerControlledNamespacedValue` to `name` automagically changes between `name` and `altName` of the underlying namespaces value, for things like Threads we only want to handle one or the other depending on which your server understands

Reply:
... that shouldn't be a NamespacedValue then, as it's breaking the contract one can expect of a namespaced value. Not worth fixing in this PR, but also not exactly happy about that being a thing.